docs/setup_selfbuild.md

```diff
@@ -1,17 +1,20 @@
 These instructions are to build and run mjolnir without using [Docker](./setup_docker.md).
-You need to have installed `yarn` 1.x and Node 16.
+To build mjolnir, you have to have installed [Node >16](https://nodejs.org/en/download/), [npm](https://docs.npmjs.com/cli/v7/configuring-npm/install) and [yarn >1.x](https://classic.yarnpkg.com/en/docs/install).
+
+Get the latest release-version https://github.com/matrix-org/mjolnir/releases/latest/ - for example `v1.6.1`.
 
 ```bash
-git clone https://github.com/matrix-org/mjolnir.git
+git clone https://github.com/matrix-org/mjolnir.git --branch v1.6.1
```

Comment:
same here, it's probably worth telling them to find the tag from the release page and keep it abstract here `<release tag>`

Reply:
Changed it to your suggestion.